File: Submissions/Educational_061/B/main.cpp
#include <bits/stdc++.h>
using namespace std;
 
int main() {
 
  int n;
  cin >> n;

  vector<int> a(n);
  long long sum = 0;
  for (int i = 0; i < n; i++) {
    cin >> a[i];
    sum += a[i];
  }

  sort(a.begin(), a.end());

  int m;
  cin >> m;

  for (int i = 0; i < m; i++) {
    int q;
    cin >> q;
    cout << sum - a[n - q] << endl;
  }

}

File: Submissions/Educational_061/B/main.java
import java.util.*;
import java.io.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();

    int[] a = new int[n];
    long sum = 0;

    for (int i = 0; i < n; i++) {
      a[i] = sc.nextInt();
      sum += a[i]; 
    }

    Arrays.sort(a);

    int m = sc.nextInt();

    PrintWriter out = new PrintWriter(System.out);
    
    for (int i = 0; i < m; i++) {
      int q = sc.nextInt();
      out.println(sum - a[n - q]);
    }

    out.flush();
    
    sc.close();
  
  }

}